content/source/docs/enterprise/private/automating-the-installer.html.md

```diff
@@ -71,9 +68,9 @@ The following apply to every installation:
   ```
 
 - `extern_vault_enable` - (Optional) Indicate if an external Vault cluster is being used. Set to `1` if so.
-  - `extern_vault_addr` - URL of external Vault cluster
-  - `extern_vault_role_id` - AppRole RoleId to use to authenticate with the Vault cluster
-  - `extern_vault_secret_id` - AppRole SecretId to use to authenticate with the Vault cluster
+  - `extern_vault_addr` - (Required) URL of external Vault cluster
+  - `extern_vault_role_id` - (Required) AppRole RoleId to use to authenticate with the Vault cluster
+  - `extern_vault_secret_id` - (Required) AppRole SecretId to use to authenticate with the Vault cluster
```

Comment:
More like

> (Required if `extern_vault_enable` is `1`)

yeah?

Reply:
I'll make that clearer with a section bullet comment, as that fact is true for every indented variable there.